CONTRIBUTING.md

```diff
@@ -0,0 +1,201 @@
+# Contributing to TiDB Dashboard
+
+Thanks for your interest in contributing to TiDB Dashboard! This document outlines some of the conventions on building, running, and testing TiDB Dashboard, the development workflow, commit message formatting, contact points and other resources.
+
+If you need any help or mentoring getting started, understanding the codebase, or making a PR (or anything else really), please ask in `#sig-dashboard` channel on [Slack](https://join.slack.com/t/tidbcommunity/shared_invite/enQtNzc0MzI4ODExMDc4LWYwYmIzMjZkYzJiNDUxMmZlN2FiMGJkZjAyMzQ5NGU0NGY0NzI3NTYwMjAyNGQ1N2I2ZjAxNzc1OGUwYWM0NzE).
+
+## Building and setting up a development workspace
+
+TiDB Dasboard is a web interface which integrate into PD by default, andd can be deploy standalone. Its back-end is written in [Golang](https://golang.org/) with [Gin](https://github.com/gin-gonic/gin), front-end is written in [Typescript](https://www.typescriptlang.org/) with [React](https://github.com/facebook/react). It uses [swaggo](https://github.com/swaggo/swag) and [openapi-generator](https://github.com/OpenAPITools/openapi-generator) for automatically generating API documents and API client. Also, to provide consistency, we use linters and automated formatting tools.
+
+### Prerequisites
+
+To build TiDB Dashboard you'll need to at least have the following installed:
+
+- `git` - Version control
+- `make` - Build tool (run common workflows)
+- [`golang`](https://golang.org/) - Golang (require 1.13+)
+- [`node.js`](https://nodejs.org/) - Javascript runtime (require 12+)
```

Comment:
Node.js

Reply:
As reported by @HunDunDM , Java is also needed for running the openapi-generator.